docs/book/address_book.rst

```diff
@@ -0,0 +1,43 @@
+.. index::
+   single: Address Book
+
+Address Book
+============
+
+The Address Book concept is a very convenient solution for the customers of your shop, that come back.
+Once they provide an address it is saved in the system and can be reused the next time.
+
+**Sylius** handles the address book in a very simple way. We are holding a collection of addresses on the Customer entity.
+
+The Addresses Collection on a Customer
+--------------------------------------
+
+On the Customer entity we are holding a collection of addresses:
+
+.. code-block:: php
+
+    /**
+     * @var Collection|AddressInterface[]
+     */
+    protected $addresses;
+
+We can operate on this collection as usually - by adding and removing objects.
+
+Besides the Customer entity has a **default address** field that is the default address used both for shipping and billing,
```

Comment:
There is no such a filed like `default address`. Customer entity has two fields which with default addresses and they are `billingAddress` and `shippingAddress`